### PR TITLE
Milan Matches Endpoint

### PR DESCRIPTION
**Pull request by Garlyle**

### CODE REVIEW

File: app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.greenfox.opal.gitinder.model.response;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+
+public class Match {
+    String username;
+    Timestamp matched_at;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java]: @Garlyle `long` or `Date`, check the json converters documentation what the project uses

File: app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    ArrayList<String> messages;
+
+    public Match(String username, Timestamp time) {
+        this.username = username;
+        matched_at = time;
+        messages = new ArrayList<>();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java]: @Garlyle no need for this type of constructor

Comment (adamgyulavari) [reply app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java]: just an empty

File: app/src/main/java/com/greenfox/opal/gitinder/ApiService.java [hunk 3]
```diff
@@ -26,5 +31,4 @@
 
   @PUT("/profiles/{username}/{direction}")
   Call<SwipingResponse> swiping(@Header(value = "X-GiTinder-token") String token, @Path("username") String username, @Path("direction") Enum<Direction> direction);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/ApiService.java]: @Garlyle check the api spec, the swiping response changed

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Review by balintvecsey — APPROVED** [review]

**Pull request merged**